app/src/common/shared/com/igalia/wolvic/ui/viewmodel/TrayViewModel.java

```diff
@@ -43,7 +45,7 @@ public TrayViewModel(@NonNull Application application) {
         isVisible.setValue(new ObservableBoolean(false));
         time = new MutableLiveData<>();
         pm = new MutableLiveData<>();
-        pm = new MutableLiveData<>();
+        needsTabsButton = new MutableLiveData<>(new ObservableBoolean(true));
```

Comment:
I know this is a bit of bikeshedding, but perhaps we could have a more descriptive name, "needsTabs" does not mean anything to me. What about something like detachedTabsButton or similar?

Reply:
How about `tabsButtonInTray`? It is simply a value to know if a tabs button should be included in the tray.